src/main/java/gim/logic/commands/AddCommand.java

```diff
@@ -27,7 +27,7 @@ public class AddCommand extends Command {
                     + "SETS " + PREFIX_REPS
                     + "REPS " + PREFIX_DATE
                     + "DATE...\n"
-                    + "Example: " + COMMAND_WORD + " " + PREFIX_NAME + "Squat " + PREFIX_WEIGHT + "60 " + PREFIX_SETS
```

Comment:
May want to consider another form of representation as a \n may look awkward, perhaps an Example -> :a ... to distinguish